MatrixSDK/Crypto/Migration/MXCryptoMigrationV2.swift

```diff
@@ -60,7 +64,69 @@ class MXCryptoMigrationV2: NSObject {
             progressListener: self
         )
         
+        log.debug("Migrating olm sessions in batches")
+        
+        // How much does migration of olm vs megolm sessions contribute to the overall progress
+        let olmToMegolmProgressRatio = 0.25
```

Comment:
We discussed this on element but yeah this could be a computed value of `olm / (olm + megolm`

Reply:
Fix applied [here](https://github.com/matrix-org/matrix-ios-sdk/pull/1692/files#diff-85e898e29304782019f3a309bd00f3f2631df8d1d0e83a306457c72c0101dae8R71)